contents/handbook/growth/sales/billing.md

```diff
@@ -17,44 +17,64 @@ To ensure consistency in the setup of annual plans we have [Zapier Automation](h
-2. Add the Stripe Customer ID (note you might need to create one and add address information if they've not previously been a customer).
+##### Upfront Payment Setup
 
-#### Create the Up-Front Invoice and Subscription Prices
+###### Step 1: Update zapier table with existing Stripe ID
 
-You can now click the [Create Annual Subscription](https://zapier.com/app/zap/206913950) button.  Using the data from the table row where the button was clicked this will:
-1. Create a draft Invoice object against the Stripe Customer Object.
-2. Add the ID of the Invoice to the table (for easy review later on).  The due date of the invoice will be the Contract Start Date + 30 days which are our standard payment terms.  **You might need to manually change this if we have different terms with the customer.**
-3. Add the up-front price for each of Product Analytics, Group Analytics, Session Replay, Feature Flags and Enterprise to the draft invoice.
-4. Where we have custom overages, create a Stripe Price for each of those and populate the table with the Price IDs.  _NOTE: We don't currently do anything here in cases where overage is none or free - need to figure out in future if this is necessary._
-5. Enable the Amortization and Schedule Subscription buttons in the table.
+If this is a new contract for an existing customer, you will need to add their existing Stripe Customer ID manually to the table. You can find this information in Vitally under Traits. If this is a brand new customer, click “Create Stripe Customer” button to assign them a new ID.
 
-#### Amortize Invoice
+###### Step 2: Create the invoice
+- Go to the [Annual Plan Table](https://tables.zapier.com/app/tables/t/01HGX2N9JXNV2EEDYARD24901R) and click “Create Invoice - Upfront”. This will:
+  - Create a draft Invoice object against the Stripe Customer Object.
+  - Add the ID of the Invoice to the table (for easy review later on). The due date of the invoice will be the Contract Start Date + 30 days which are our standard payment terms. You might need to manually change this if we have different terms with the customer.
 
-As Zapier works asynchronously we need to wait until the table is updated with all of the annual prices (indicating that the invoice is ready).  Clicking the [Amortize Invoice](https://zapier.com/app/zap/207286989) button will set the correct metadata for our revenue tracking on the invoice.
+###### Step 3: Verify invoice details and send
+- Use the Invoice ID recorded in the table to locate the invoice in Stripe.
+- Ensure all details are correct, particularly the Customer’s Billing/Shipping addresses and Tax ID on the Customer object.
+- Send the invoice to the customer and wait for the payment to be completed. Do not proceed to the next steps until payment is confirmed.
 
-#### Schedule Subscription
+###### Step 4: Apply credit
+- Once payment is confirmed, return to the Zapier table and click the “Apply Credit” button to add purchased credits to the customer account.
```

Comment:
This should only happen immediately if they are using credits for the current cycle. As soon as someone has credits on their account, they will get used for invoices. So this credit apply needs to be scheduled somehow if they don't want to use the credits for the current bill cycle.

Reply:
yep that's mentioned at the end of step 3 but I'll make it more obvious and clarify for monthly setup as well

Reply:
updated!